bytecomp/lambda.ml

```diff
@@ -264,9 +264,25 @@ and lambda_event_kind =
   | Lev_function
   | Lev_pseudo
 
+type module_map = {
+  map :  (int * Ident.t) list;
```

Comment:
This cannot represent the flambda version, it needs to be `(symbol * int)`.
This is the same for methcache.
It means that cmmgen cannot use global_data to emit that.